Source/RapyutaSimulationPlugins/Public/Robots/RRBaseRobot.h

```diff
@@ -57,6 +57,22 @@ class RAPYUTASIMULATIONPLUGINS_API ARRBaseRobot : public ARRBaseActor
     //! Robot's ROS2 Interface
     UPROPERTY(Replicated)
     URRRobotROS2Interface* ROS2Interface = nullptr;
+
+    //! is this robot is possed or not. Defaults method's do not works with multiplayer.
+    UPROPERTY(Replicated)
+    bool isPossessed = false;

```

Comment:
`b` prefix for boolean
```suggestion
    //! Whether this robot is possessed or not. `GetController()` do not work with multiplayer.
    UPROPERTY(Replicated)
    bool bIsPossessed = false;
```
Just to confirm, [here](https://github.com/EpicGames/UnrealEngine/blob/release/Engine/Source/Runtime/Engine/Private/Pawn.cpp#L559) it seems `APawn::Controller` does get replicated?
Also `APawn::GetLifetimeReplicatedProps()` has it fetched. @larryng-rapyuta

Reply:
I did experienced  like this just inside FindPawnToPossess, and result is 0 1 1 just after posses by one of networkplayer
```
            UE_LOG(LogRapyutaCore,
                   Error,
                   TEXT("Player [%s] %d %d %d"),
                   *PlayerName,
                   !robot->IsPossessed(),
                   nullptr == robot->GetController(),
                   nullptr == robot->Controller);

Reply:
@larryng-rapyuta?

Reply:
Yeah, it looks like controller is not really replicated. This may give some insight
https://forums.unrealengine.com/t/in-multiplayer-mode-cant-get-other-characters-player-id-of-other-clients-in-current-client/578498/7

They recommend `PlayerState` instead, could you try using it?
https://github.com/EpicGames/UnrealEngine/blob/release/Engine/Source/Runtime/Engine/Classes/GameFramework/Pawn.h#L153

Reply:
Thanks, it works